src/Cache/DirectoryInterface.php

```diff
@@ -20,4 +20,6 @@
 interface DirectoryInterface
 {
     public function getRelativePathTo(string $file): string;
+
+    public function getAbsolutePath(): string;
```

Comment:
Isn't adding a method here a BC break?

Reply:
I was pretty sure it is internal interface... but it seems only its implementation is marked as `@internal` 🤔. I'll look at it, maybe I can achieve this differently.

Reply:
This change was reverted, I was able to handle it differently - pass absolute path from the worker and trim it to relative path on the runner side.